lightning/src/ln/channel.rs

```diff
@@ -3068,21 +3097,23 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 				propose_new_feerate!(max_feerate);
 			}
 		} else {
-			let min_feerate = fee_estimator.get_est_sat_per_1000_weight(ConfirmationTarget::Background);
-			if (proposed_sat_per_kw as u32) < min_feerate {
-				if let Some((last_feerate, _, _)) = self.last_sent_closing_fee {
-					if min_feerate >= last_feerate {
-						return Err(ChannelError::Close(format!("Unable to come to consensus about closing feerate, remote wanted something lower ({}) than our Background feerate ({}).", last_feerate, min_feerate)));
-					}
+			min_feerate = fee_estimator.get_est_sat_per_1000_weight(ConfirmationTarget::Background);
+		}
+		if (msg.fee_satoshis as u64) < min_feerate as u64 * closing_tx_max_weight / 1000 {
+			if let Some((last_feerate, _, _)) = self.last_sent_closing_fee {
+				if min_feerate >= last_feerate {
```

Comment:
It should be `>`, not `>=`?

Reply:
If its ==, that means the last time we proposed something to our counterparty it was already `min_feerate` and they came back with a counter-proposal. Proposing again would get us into a loop.